packages/voter-dapp/src/DesignatedVotingDeployment.js

```diff
@@ -53,7 +53,11 @@ function DesignatedVotingDeployment({ votingAccount }) {
       <strong>The two key voting contract is completely optional.</strong>
       <br />
       To deploy one, enter your cold key address. To learn more about the 2 key voting system see our{" "}
-      <a href="https://docs.umaproject.org/uma/oracle/voting_with_UMA_2-key_contract.html" target="_blank">
+      <a
+        href="https://docs.umaproject.org/uma/oracle/voting_with_UMA_2-key_contract.html"
+        rel="noopener noreferrer"
```

Comment:
this was added to stop compiler warning

Reply:
Why does the `rel` fix the compiler warning OOC?

Reply:
![image](https://user-images.githubusercontent.com/4429761/109701524-1dfb8880-7b61-11eb-98e2-6af29b5cd2aa.png)